### PR TITLE
move staging to new rabbit MQ.

### LINKED ISSUE #401: [TASK]: move staging to new RabbitMQ
**Describe the task**
A clear and concise description of what the task is.

**Task Items**
- [ ] list of items for completion
- [ ] another item

**Goal**
A clear and concise description of what the end result must have for it to be approved as complete.

**Screenshots**
If applicable, add screenshots or designs

**Additional context**
Add any other context about the task here

### PR DESCRIPTION
**Pull request by dgauldie**
# Description

Please include a summary of the changes and the related issue.

Resolves #401

### CODE REVIEW

File: kubernetes/overlays/prod/overlays/askem-production/services/pyciemss-service/pyciemss-worker-deployment.yaml [hunk 1]
```diff
@@ -13,7 +13,5 @@ spec:
               value: redis.terarium.ai
             - name: REDIS_PORT
               value: '6379'
-            - name: RABBITMQ_HOST
-              value: 'b-68052fee-712b-461b-81c3-7a82a8112d50.mq.us-east-1.amazonaws.com'
-            - name: RABBITMQ_PORT
-              value: '5671'
+            - name: RABBITMQ_SSL
+              value: 'True'
```
Comment (YohannParis) [inline kubernetes/overlays/prod/overlays/askem-production/services/pyciemss-service/pyciemss-worker-deployment.yaml:17]: Why is it `True` and not `true` like in the `sciml-service` for example?

File: kubernetes/overlays/prod/overlays/askem-staging/services/pyciemss-service/pyciemss-api-deployment.yaml [hunk 2]
```diff
@@ -13,7 +13,5 @@ spec:
               value: redis.staging.terarium.ai
             - name: REDIS_PORT
               value: '6379'
-            - name: RABBITMQ_HOST
-              value: 'b-1cb71505-9454-4d81-8ca0-a0ee6faf5770.mq.us-east-1.amazonaws.com'
-            - name: RABBITMQ_PORT
-              value: '5671'
+            - name: RABBITMQ_SSL
+              value: 'False'
```
Comment (YohannParis) [inline kubernetes/overlays/prod/overlays/askem-staging/services/pyciemss-service/pyciemss-api-deployment.yaml:17]: Same question, with `False` and `false`.

**Review by YohannParis — APPROVED** [review]

File: kubernetes/overlays/prod/overlays/askem-production/services/pyciemss-service/pyciemss-worker-deployment.yaml (hunk 1 above)
Comment (dgauldie) [reply kubernetes/overlays/prod/overlays/askem-production/services/pyciemss-service/pyciemss-worker-deployment.yaml:17]: python vs julia

File: kubernetes/overlays/prod/overlays/askem-staging/services/pyciemss-service/pyciemss-api-deployment.yaml (hunk 2 above)
Comment (dgauldie) [reply kubernetes/overlays/prod/overlays/askem-staging/services/pyciemss-service/pyciemss-api-deployment.yaml:17]: python vs julia

**Pull request merged**